### PR TITLE
Feature/jenkins

### PR DESCRIPTION
**Pull request by mrIncompetent**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,7 @@
-FROM alpine:3.1
+FROM alpine:3.5
 MAINTAINER Dr. Stefan Schimanski <stefan.schimanski@gmail.com>
```
Comment (scruplelesswizard) [inline Dockerfile]: [Maintainer is Deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).

Please drop or use [Labels](https://docs.docker.com/engine/reference/builder/#label) instead

File: Makefile [hunk 2]
```diff
@@ -22,35 +22,33 @@ gofmt:
...
 
 clean:
 	rm -f $(CMD)
 
 install:
 	glide install --strip-vendor
 
-docker: $(CMD)
-	@if [ "$$GOOS" != linux ]; then echo "Run make with GOOS=linux"; exit 1; fi
+docker:
```
Comment (scruplelesswizard) [inline Makefile]: please rename to `docker-build`

File: Makefile [hunk 3]
```diff
@@ -22,35 +22,33 @@ gofmt:
...
 	docker build -t $(REPO) .
 
 push: docker
```
Comment (scruplelesswizard) [inline Makefile]: Please rename to `docker-push`

**Review by scruplelesswizard — CHANGES_REQUESTED** [review]

**Review by scruplelesswizard — CHANGES_REQUESTED** [review]
PR is missing documentation

**Comment by mrIncompetent** [comment]
@chaosaffe `README.md` has been updated to reflect the current changes

**Pull request merged**